ansible/configs/sap-integration/post_infra.yml

```diff
@@ -0,0 +1,22 @@
+- name: Step 002 - Post Infrastructure
+  hosts: localhost
+  connection: local
+  become: false
+  tags:
+  - step002
+  - post_infrastructure
+  tasks:
+  - name: OpenStack Post Infrastructure
+    when: cloud_provider is match("osp")
+    environment:
```

Comment:
These should not be necessary when you are only running that single role.

Reply:
done